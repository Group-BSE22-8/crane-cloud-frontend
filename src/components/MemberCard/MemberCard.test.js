import React from "react";
import { shallow } from "enzyme";
import MemberCard from ".";

describe('Testing the MemberCard component', () => {
    it('should match the member card snapshot', () => {
        const MemberCardComponent = shallow(<MemberCard  name="" title="" icon=""  />);
        expect(MemberCardComponent).toMatchSnapshot();
    })
})