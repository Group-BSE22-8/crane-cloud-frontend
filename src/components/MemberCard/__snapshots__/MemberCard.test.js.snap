// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the MemberCard component should match the member card snapshot 1`] = `ShallowWrapper {}`;
